vitest.config.ts

```diff
@@ -6,6 +6,7 @@ export default defineConfig({
     environment: 'jsdom',
     setupFiles: 'test-setup.ts',
     clearMocks: true,
+    fileParallelism: false,
```

Comment:
True by default was causing issues when using github actions; some tests were working if you run them individually but, some failed if you run `npm run test`

Reply:
If tests are working when you run them individually but not in the full suite it is probably pointing to something not being cleaned up correctly between the tests